worlds/wargroove2/RegionFilter.py

```diff
@@ -0,0 +1,23 @@
+class Wargroove2LogicFilter:
+    items: [str]
```

Comment:
Assuming this is typing (and a few other places in this file)
```suggestion
    items: List[str]
```